com.woltlab.wcf/templates/boxRecentActivity.tpl

```diff
@@ -44,6 +44,6 @@
 		{jsphrase name='wcf.user.recentActivity.more'}
 		{jsphrase name='wcf.user.recentActivity.noMoreEntries'}
 
-		setup(document.getElementById('boxRecentActivity{unsafe:$boxID|encodeJS}'));
```

Comment:
This change is incorrect, the rules for encoding values in JS strings differ from HTML. Please revert this back to the original, correct way of handling the value.